### ISSUE TITLE
Stop does not clean registred stream 

### ISSUE THREAD

**Issue by alborq** [sunli829/xactor#14]
When i stop an actor registred stream keep alive

Exemple with given Actor : 
```rust 
use std::time::Duration;
use tokio::io;
use tokio::net::{TcpStream, TcpListener};
use xactor::*;

struct MyActor;

#[async_trait::async_trait]
impl Actor for MyActor {
    async fn started(&mut self, ctx: &Context<Self>) {
        let listener = TcpListener::bind("0.0.0.0:1234").await.unwrap();
        ctx.add_stream(listener);
        ctx.stop(None);
    }
}

#[async_trait::async_trait]
impl StreamHandler<io::Result<TcpStream>> for MyActor {
    async fn handle(&mut self, _ctx: &Context<Self>, inbound: io::Result<TcpStream>) {
        println!("echo !");
    }
}

#[xactor::main]
async fn main() {
    println!("Start");
    // Exit the program after 3 seconds
    let addr = MyActor.start().await;
    addr.wait_for_stop().await;
    println!("Actor stopped");
    
    sleep(Duration::from_secs(3600)).await;
}
```

I cant fix myself :( 
- I dont know how wait stream or onechannel on same time and resolve first 
- Actix choose another solution and encapsulate Stream in a dedicated sub-actor

**Comment by sunli829** [comment]
Released `v0.7.2`.

**Comment by alborq** [comment]
Work perfectly
